README.md

```diff
@@ -11,6 +11,19 @@ computing.
 
 ![Jupyter notebook example](docs/resources/running_code_med.png "Jupyter notebook example")
 
+Since the release of JupyterLab, many users of the classic Jupyter Notebook have been migrating to using the new front-end.
```

Comment:
This opening sentence feels a little out of place.  I wonder if it should be included in the paragraph below?  

FWIW, the pending 6.1 changelog currently uses this kind of language: 
>*Please note that this repository is currently maintained by a skeleton crew of maintainers from the Jupyter community. As a result, we encourage users to transition to JupyterLab, where more immediate support can occur.*

Or we could drop it allowing the `Alternatives to contributing...` paragraph to serve its purpose.